README.md

```diff
@@ -90,6 +90,13 @@ expander.addEventListener('text-expander-value', function(event) {
 })
 ```
 
+**`text-expander-dismiss`** is an event the text-expander-element is listening to. The event can be fired in order to hide the menu provided through `provide` method. It is not expecting any parameter. It is used internally to hide the menu when Escape key it hit.
```

Comment:
> It is used internally to hide the menu when Escape key it hit.

I am confused by this sentence. AFAIK `text-expander-element` already dismisses menu on `Esc` key so this event is not needed for that. My understanding was that this event was introduced to dismiss the menu by some external factors. Should we get rid of that sentence?

```suggestion
**`text-expander-dismiss`** is an event the text-expander-element is listening to. The event can be fired in order to hide the menu provided through `provide` method. It is not expecting any parameter.
```

Reply:
Do you mean expose through this text or in the code? 
If you mean through the text, & you are referring to `It is used internally to hide the menu when Escape key it hit` : not sure it is needed, i did it mainly because it is stopping propagation of Esc and this could be useful information for other users of this lib

Reply:
resolving this as i changed the implementation and removed the dismiss event